eventmesh-admin/src/main/java/org/apache/eventmesh/admin/common/NacosConst.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.admin.common;
+
+public class NacosConst {
+
+    public static final String LOGIN_API = "/nacos/v1/auth/login";
+
+    public static final String LOGIN_REQ_USERNAME = "username";
+    public static final String LOGIN_REQ_PASSWORD = "password";
+
+    public static final String LOGIN_RESP_TOKEN = "accessToken";
+
+    public static final String CONFIGS_API = "/nacos/v1/cs/configs";
+
+    public static final String CONFIGS_REQ_PAGE = "pageNo";
+    public static final String CONFIGS_REQ_PAGE_SIZE = "pageSize";
+    public static final String CONFIGS_REQ_DATAID = "dataId";
+    public static final String CONFIGS_REQ_GROUP = "group";
+    public static final String CONFIGS_REQ_SEARCH = "search";
+    public static final String CONFIGS_REQ_TOKEN = "accessToken";
+
+    public static final String CONFIGS_RESP_CONTENT_LIST = "pageItems"; // json page data list field
+    public static final String CONFIGS_RESP_CONTENT = "content"; // json page data field
+    public static final String CONFIGS_RESP_PAGES = "pagesAvailable"; // json total pages field
+    public static final String CONFIGS_RESP_DATAID = "dataId";
+    public static final String CONFIGS_RESP_GROUP = "group";
```

Comment:
Yes, because their "meanings" belong to the request body and the response body respectively, it's just that they **happen to** “share” the same name. If the field names in the request body are changed, and the field names in the response body remain unchanged, then the field names in both the request body and the response body should not be modified at the same time.

---

是的，因为它们的“含义”分别属于请求体和响应体，只不过“恰好”重名了而已。如果请求体中的字段名称发生了更改，而响应体中的字段名称没有发生更改，那么就不应该同时修改请求体和响应体的字段名称。

Reply:
This depends on the scope of the field's impact. If the field is a "concept" for the entire project, then defining it in multiple places naturally does not make it easy to synchronise changes. I think this is the case with eventmesh's protocol. But #4474 has some other problems: https://github.com/apache/eventmesh/pull/4495#pullrequestreview-1689037870

---

这取决于字段的影响范围。如果这个字段是整个项目的一个“概念”，那定义在多个地方自然不便于同步修改。我觉得eventmesh的协议属于这种情况。但是#4474 还存在一些其它问题：https://github.com/apache/eventmesh/pull/4495#pullrequestreview-1689037870